app/views/feedback/admin.html.erb

```diff
@@ -13,11 +13,15 @@
       <tbody>
         <% @feedback.each do |feedback| %>
           <tr>
-            <td><%= feedback.username %></td>
-            <td><%= feedback.created_at.strftime("%m/%d/%Y") %></td>
-            <td><%= feedback.application %></td>
-            <td><%= feedback.feedback %></td>
-            <td>
+            <% if feedback.contact_email.blank? %>
+              <td valign="top"><%= feedback.username %></td>
```

Comment:
Looks like this works. But it's a common practice to separate styling/positioning from HTML. if you add 

```
.align-top {
  vertical-align: top
}
```

in `_main.scss` then add `class="align-top"` to each of these tds, that should do it!